### PR TITLE
Add release configuration (fixes #48)

### LINKED ISSUE #48: Release to bintray

### PR DESCRIPTION
**Pull request by baev**
Maven artefacts are available here - https://dl.bintray.com/qameta/maven/
Allure CLI and plugins - https://dl.bintray.com/qameta/generic/

### CODE REVIEW

File: gradle/bintray.gradle [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        repo = 'maven'
+        name = 'allure2'
+        desc = 'Allure Report'
+        websiteUrl = 'https://github.com/allure-framework/allure2'
+        issueTrackerUrl = 'https://github.com/allure-framework/allure2'
+        vcsUrl = 'https://github.com/allure-framework/allure2.git'
+        licenses = ['Apache-2.0']
+
+        githubRepo = 'allure-framework/allure2'
+        githubReleaseNotesFile = 'README.md'
```
Comment (eroshenkoam) [inline gradle/bintray.gradle]: Нужно в будущем генерировать релизноутсы)

File: settings.gradle [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 include 'allure-plugin-api'
+include 'allure-generator'
+include 'allure-web'
 include 'behaviors-plugin'
 include 'custom-logo-plugin'
 include 'junit-plugin'
 include 'packages-plugin'
+include 'allure-commandline'
 
 project(':behaviors-plugin').projectDir = new File(settingsDir, 'plugins/behaviors-plugin')
```
Comment (lanwen) [inline settings.gradle]: JFMI - can u explain what it means?

Comment (baev) [inline settings.gradle]: plugins located in subdirectory called `plugins`. So this code changes module location for behaviors-plugin from `behaviors-plugin` to 'plugins/behaviors-plugin'

Comment (baev) [inline settings.gradle]: you can not use `include 'plugins/behaviors-plugin'` because in that case module gonna be called `plugins/behaviors-plugin`

Comment (lanwen) [inline settings.gradle]: it can't be written as simple `include plugins/behaviors-plugin`?

Comment (lanwen) [inline settings.gradle]: https://docs.gradle.org/current/dsl/org.gradle.api.initialization.Settings.html
hmm, u can add it as `include ':plugins:behaviours-plugin'` and then change name as `project(':plugins:behaviors-plugin').name = ':behaviors-plugin'`

which looks more accurate

Comment (lanwen) [inline settings.gradle]: (most of people don't rename anything affter declaration)

**Pull request merged**